CHANGELOG.md

```diff
@@ -88,6 +145,16 @@ These functions and structs can be referenced in both code and blueprints and it
 - Unreal Engine version 4.26.0 is supported! Refer to https://documentation.improbable.io/gdk-for-unreal/docs/keep-your-gdk-up-to-date for versioning information and how to upgrade.
 - Running with an out-of-date schema database reports a version warning when attempting to launch in editor.
 - Reworked schema generation (incremental and full) pop-ups to be clearer. 
+- Unreal Engine version 4.26.0 is now supported! Refer to https://documentation.improbable.io/gdk-for-unreal/docs/keep-your-gdk-up-to-date for versioning information and how to upgrade.
```

Comment:
Hmm, this looks wrong.
1/ Lots of duplicate lines
2/ Why are we modifying the 0.12 release notes at all in the 0.13 release?

Reply:
After a lot seartching around we've found out the reason why. 
Commits were made to the 0.12.0-rc-cleanup branch which is the post release merge back into master.
These commits were not actually already in the `release` branch and modified the 0.12 release notes.
This is dangerous and annoying and has caused significant pain. Anyway, we're going to de-dupe these and cleanup.
Thanks for spotting dude. 

https://github.com/spatialos/UnrealGDK/pull/2874/commits/d86dfa2c5437ee6a8e00cfef74405fb33d30cffd
https://github.com/spatialos/UnrealGDK/pull/2874/commits/eb2413beed12f1d17c7a1c8440cc9bad6f1931c4